colin-api/src/colin_api/models/filing.py

```diff
@@ -796,6 +808,11 @@ def add_filing(cls, con, filing):  # pylint: disable=too-many-locals,too-many-st
         try:
             corp_num = filing.get_corp_num()
             legal_type = corp_num[:2]
+
+            if (legal_type!='CP'):
+               # Future: May need a different way of determining legal type
+               legal_type = 'BC'
```

Comment:
Shouldn't this be 'BEN'?

Reply:
When I was testing this with David I had to update this in a few places

Reply:
D'oh